util/cron/nightly

```diff
@@ -152,6 +158,8 @@ while (@ARGV) {
         $asserts = 1;
     } elsif ($flag eq "-mason") {
         $mason_build = 1;
+    } elsif ($flag eq "-chapcs") {
```

Comment:
I'd probably call this `paratest.chapcs` and I'm thinking we'll want to be able to specify the `-nodepara` so that we can still increase the amount of oversubscription we're using for valgrind.

Reply:
Just so I'm clear: You are suggesting that we add `-nodepara` to both `nightly` and `paratest.chapcs` and make that override `get_good_nodepara` in `paratest.chapcs`, right?

Reply:
Yeah, I would have nightly take a `-paratest.chapcs <nodepara>` flag and pass `-nodepara <nodepara>` to paratest.chapcs (If you just specify `-nodepara <nodepara>` to paratest.chapcs it will override whatever value paratest.chapcs calculates.